e2e/production-e2e/Dockerfile

```diff
@@ -20,38 +17,43 @@ RUN apt-get install -y libwoff1 \
                        libxslt1.1 \
                        libevent-2.1-6 \
                        libgles2 \
-                       libvpx5
-
-# 3. Install Chromium dependencies
+                       libvpx5 
 
+# 2. Install Chromium dependencies
 RUN apt-get install -y libnss3 \
                        libxss1 \
                        libasound2
 
-# 4. Install Firefox dependencies
-
+# 3. Install Firefox dependencies
 RUN apt-get install -y libdbus-glib-1-2
 
-# 5. Add user so we don't need --no-sandbox in Chromium
+# 4. Add user so we don't need --no-sandbox in Chromium
 # RUN groupadd -r pwuser && useradd -r -g pwuser -G audio,video pwuser \
 #     && mkdir -p /home/pwuser/Downloads \
 #     && chown -R pwuser:pwuser /home/pwuser
 
-# 6. (Optional) Install XVFB if there's a need to run browsers in headful mode
+# 5. (Optional) Install XVFB if there's a need to run browsers in headful mode
 RUN apt-get install -y xvfb
 
+# Remove package manager cache
+RUN rm -rf /var/lib/apt/lists/*
+
+# Create non-root node user
+RUN groupadd --gid 1000 node \
```

Comment:
Why use has name node? Try to use the unique name to avoid conflict with base images, maybe hydro? We need to meet about with